Assets/__Scripts/Beatmap/Containers/NoteContainer.cs

```diff
@@ -11,6 +13,7 @@ public class NoteContainer : ObjectContainer
 
         [SerializeField] private GameObject simpleBlock;
         [SerializeField] private GameObject complexBlock;
+        [SerializeField] public Transform directionTarget;
```

Comment:
me when `.editorconfig` violation (you'll have to use `FormerallySerializedAs` attribute to not break references in the Unity prefab/scene)

Reply:
*My boring text editor doesn't understand .editorconfig and neither do I*
*Also there's no old name so what would `FormerallySerializedAs` point to?*

Reply:
[`.editorconfig`](https://editorconfig.org/) enforces code styling solution-wide. Makes the code consistent, even when multiple people work on the project. Moreover, because I'm an evil person and want the codebase to at least *look* consistent, [incoming PRs are required to follow `.editorconfig`](https://github.com/Caeden117/ChroMapper/blob/master/CONTRIBUTING.md#coding-convention--styling-guidelines).

Currently, the Unity prefab is looking for `directionTarget` to assign the Transform to. After `.editorconfig` violations have been fixed, this association is broken. You use the `[FormerallySerializedAs]` attribute with the old name of the field:

```cs
[SerializeField, FormerallySerializedAs("directionTarget")] public Transform DirectionTarget;
```

Reply:
So is the problem that public members must be capitalized? I got a trail of Jetbrains Rider and it still isn't telling me anythings wrong.

Reply:
yes. interestingly, Rider should be picking up `.editorconfig` automatically.

Reply:
The rule is there, it just doesn't show any error or warning.
![image](https://github.com/Caeden117/ChroMapper/assets/74876786/77a3efe4-6ddd-4576-b47b-77411b4b3d0d)

Made sure this was turned on too:
![image](https://github.com/Caeden117/ChroMapper/assets/74876786/7dcdd230-c760-4db5-b09e-2024c69553d4)